statics/js/components/topology.js

```diff
@@ -27,8 +27,6 @@ var TopologyComponent = {
           </button>\
           <button id="zoom-reset" type="button" class="btn btn-primary" \
```

Comment:
It is not related to your patch but I think we will have too much button and this one seems to be useless. The "fit" one is quite similar and more useful. Maybe it is time to remove it ?

Reply:
yes. will remove it.